waggle-dance/src/main/assembly/waggle-dance-package.xml

```diff
@@ -16,10 +16,10 @@
   <dependencySets>
     <dependencySet>
       <outputDirectory>service</outputDirectory>
-      <outputFileNameMapping>${artifact.artifactId}-latest${dashClassifier?}.${artifact.extension}</outputFileNameMapping>
+      <outputFileNameMapping>waggle-dance-core-latest${dashClassifier?}.${artifact.extension}</outputFileNameMapping>
```

Comment:
Anyway we could achieve this with symlinks for backwards compatability with the old "core" names but pointing to files with names not containing core (the "boot" files)?

Reply:
not really. can't do symlinks in assembly

Reply:
Maven Ant plugin here we come!

Reply:
the name will be `waggle-dance-core-latest-exec.jar` not the same as the normal "core" jar it will have the exec classifier. I agree not ideal to have "core" in the name but I rather not clutter it all up with symlinks and just keep it backward compatible.

Reply:
OK.